ntbs-ui-tests/Features/EditNotifications.feature

```diff
@@ -0,0 +1,324 @@
+Feature: Notification editing
+  Happy and error paths for notification creation
+  Notification deletion
+
+  Background: Create new notification
+    Given I have logged in as ServiceUser
+    Given I am on seeded 'TO_BE_EDITED' notification overview page
+
+  Scenario: Edit notification patient details fields
+    When I go to edit the 'PatientDetails' section
+    Then I should be on the PatientDetails page
+    When I enter Jester into 'PatientDetails_GivenName'
+    And I enter Leicester into 'PatientDetails_FamilyName'
+    And I enter 4 into 'FormattedDob_Day'
+    And I enter 8 into 'FormattedDob_Month'
+    And I enter 1932 into 'FormattedDob_Year'
+    And I select radio value 'sexId-2'
+    And I enter 9866698 into 'PatientDetails_LocalPatientId'
+    And I select Chinese for 'PatientDetails_EthnicityId'
+    And I select radio value 'nhs-number-known'
+    And I enter 9452651220 into 'PatientDetails_NhsNumber'
+    And I select Other from input list 'PatientDetails_CountryId'
+    And I enter 1955 into 'year-of-entry'
+    And I enter 56 Schnowzer Boulevard into 'PatientDetails_Address'
+    And I select radio value 'postcode-yes'
+    And I enter ST1 1AA into 'PatientDetails_Postcode'
+    And I make selection 26 from Other section for 'PatientDetails_OccupationId'
+    When I click on the 'save-button' button
+
+    Then I can see the value 'LEICESTER, Jester' for the field 'full-name' in the 'PatientDetails' overview section
+    And I can see the value 'Female' for the field 'sex' in the 'PatientDetails' overview section
+    And I can see the value '945 265 1220' for the field 'nhs-number' in the 'PatientDetails' overview section
+    And I can see the value 'Retired' for the field 'occupation' in the 'PatientDetails' overview section
+    And I can see the value '04 Aug 1932' for the field 'dob' in the 'PatientDetails' overview section
+    And I can see the value '56 Schnowzer Boulevard' for the field 'address' in the 'PatientDetails' overview section
+    And I can see the value 'ST1 1AA' for the field 'postcode' in the 'PatientDetails' overview section
+    And I can see the value 'Chinese' for the field 'ethnicity' in the 'PatientDetails' overview section
+    And I can see the value 'Other' for the field 'country' in the 'PatientDetails' overview section
+    And I can see the value '1955' for the field 'uk-entry' in the 'PatientDetails' overview section
+    And I can see the value '9866698' for the field 'local-patient-id' in the 'PatientDetails' overview section
+
+  Scenario: Edit notification hospital details fields
+    When I go to edit the 'HospitalDetails' section
+    Then I should be on the HospitalDetails page
+    When I enter 2 into 'FormattedNotificationDate_Day'
+    And I enter 2 into 'FormattedNotificationDate_Month'
+    And I enter 2011 into 'FormattedNotificationDate_Year'
+      # Wait until javascript has populated the hospital dropdown
+    And I wait
```

Comment:
I thought that our implicit wait would cover a scenario like this... was this test failing before you added the manual wait?

Reply:
This was here before, but I haven't tried removing it so I'll try that and see if we actually need it.